src/components/CardMenus/CommonCard.js

```diff
@@ -40,39 +45,82 @@ const useStyles = makeStyles((theme) => ({
   expandOpen: {
     transform: 'rotate(180deg)',
   },
-  avatar: {
-    backgroundColor: 'red',
-  },
+
   image: {
     padding: '30px',
-    border: '10px',
-    borderRadius: '200px',
+    border: '100px',
+    boxShadow: '10px 10px F0F0F0',
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+  },
+  Price: {
+    paddingTop: '10px',
+    paddingLeft: '50px',
+    fontSize: '18px',
+    fontWeight: '700',
+  },
+
+  btn: {
+    marginTop: '10px',
+    minWidth: '10px',
   },
 }));
-const CommonCard = ({ name, price, resturantName, img }) => {
+const CommonCard = ({ name, price, resturantName, img, buttonText }) => {
```

Comment:
Do you investigate that we need the card in other place ??
If yes then it is ok to keep it in common .. If it is specific to feature it should be in the specific feature directory ..